src/library/baseplaylistfeature.cpp

```diff
@@ -675,3 +713,21 @@ void BasePlaylistFeature::slotTrackSelected(TrackPointer pTrack) {
 void BasePlaylistFeature::slotResetSelectedTrack() {
     slotTrackSelected(TrackPointer());
 }
+
+QString BasePlaylistFeature::getPlaylistFile() {
+    QString lastPlaylistDirectory = m_pConfig->getValueString(
```

Comment:
We should avoid duplicated code. Please move this to the parent class if possible.